### PR TITLE
Feature/improvement card list

### PR DESCRIPTION
**Pull request by cinthia-neko**
- Add LineList component.
- Add auto loading functionality to LineList.

### CODE REVIEW

File: src/components/LineList/LineList.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    super(props);
+
+    this.state = {
+      loader: false
+    };
+
+    this.onBottomVisible = this.onBottomVisible.bind(this);
+  }
+
+  onBottomVisible(){
```
Comment (ougarte) [inline src/components/LineList/LineList.js]: I think, there is not an space...

File: src/components/LineList/LineList.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import Loader from '../Loader/Loader';
+import './LineList.css';
+
+class LineList extends React.Component {
+
+  constructor(props){
+    super(props);
+
+    this.state = {
+      loader: false
```
Comment (ougarte) [inline src/components/LineList/LineList.js]: is this `loader` means `loading` ? if so, I recommend to move to props, and should be provided by the component that is using this component

File: src/components/LineList/LineList.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    };
+
+    this.onBottomVisible = this.onBottomVisible.bind(this);
+  }
+
+  onBottomVisible(){
+    if(this.props.onNextPage && !this.state.loader && this.props.nextPage) {
```
Comment (ougarte) [inline src/components/LineList/LineList.js]: [no blocker] I think, it is fine to validate onNextPage to executed or not... but I am not sure about why the others are required. Could you explain us a little, please ?

File: src/components/LineList/LineList.js (hunk 2 above)
Comment (duhnnie) [reply src/components/LineList/LineList.js]: This is not a component, this is a flag to keep track (internal state) if the loader was already displayed.

File: src/components/LineList/LineList.js (hunk 1 above)
Comment (cinthia-neko) [reply src/components/LineList/LineList.js]: Done

File: src/components/LineList/LineList.js (hunk 2 above)
Comment (cinthia-neko) [reply src/components/LineList/LineList.js]: it was removed to props

File: src/components/LineList/LineList.js (hunk 3 above)
Comment (cinthia-neko) [reply src/components/LineList/LineList.js]: they were removed nextPage and loader, they were unnecessary.

**Pull request merged**